source/php/Component/Acceptance/acceptance.blade.php

```diff
@@ -0,0 +1,57 @@
+<!-- acceptance.blade.php -->
+<div class="{{ $class }} js-suppressed-content js-suppressed-content{{$modifier}}" {!! $attribute !!}>
```

Comment:
Question: Does the BEM standard apply to _all_ classes added to the elements or only the ones actually used in the stylesheets (not js) @sebastianthulin?

Reply:
Above is fine. Behavoiur (js) do no need to be BEM-ified.

Reply:
This was a thought I had as well, I would prefer keeping the JS classes separate from the styling. To increase the usage cases for the component we will have to in some way distinguish between which component is being hidden. 
It has been moved to backend through classList :)

Reply:
Ill leave this one open